tasks/build.js

```diff
@@ -27,18 +27,18 @@ bozon.task('scripts:renderer', function () {
 });
 
 bozon.task('config', function () {
-  bozon.src('package.json').pipe(jsonEditor({
+  return bozon.src('package.json').pipe(jsonEditor({
     settings: settings.get()
   })).pipe(bozon.dest());
 });
 
 bozon.task('images', function () {
-  bozon.src('images/**/*').pipe(bozon.dest('images'));
+  return bozon.src('images/**/*').pipe(bozon.dest('images'));
 });
 
-bozon.task('compile', function () {
+bozon.task('compile', function (callback) {
   settings = new bozon.Settings();
-  runSequence('scripts:main', 'scripts:renderer', 'styles', 'html', 'images', 'config');
+  runSequence.apply(this, ['scripts:main', 'scripts:renderer', 'styles', 'html', 'images'].concat(bozon.hooks, 'config', callback));
```

Comment:
If you would like a taskplacement I can add one.

I would prefere your second example. Something like:

- `bozon.buildTask(...)` - add the task to the end of pipeline (after config)
- `bozon.buildTaskAfter('x', 'taskname', ...)` - add the task after task "x"
- `bozon.buildTaskBefore('x', 'taskname', ...)` - add the task before task "x"

If "x" is not defined then default to add the task to the end.

Is this ok for you?